### PR TITLE
Got Service Providers Rendering In Log A Service Form

### PR DESCRIPTION
**Pull request by ahmedseragcodes**
### Description

**What work was done?**
-Fixed get service providers action in serviceActions as well as property names in addAServiceForm component that mapped through service providers, so now data is being properly pulled into log a service form 
-fixed width of date column in services table and moved to front of table for better UX / organization 

**Why was this work done?**

We fixed service providers because that data is necessary to properly filling out the log a service form, as well as to ensure our backends/frontends are communicating properly.

We made changes to the services table to improve the user experience of our users when analyzing services, they would want to be able to know what date they're looking at right away. 

- What is the purpose of the feature?

To seamlessly fill out the log a service form when on-site assisting someone in need as well as to be able to easily understand what services have been rendered over a specific time period.

**What feature / user story is it for?**

https://trello.com/c/CE8PYkjA

**Any relevant links or images / screenshots**

https://lambda-students.slack.com/archives/C02AFKQBD2L/p1628624807174800

### Type of change
- Fixing existing feature (non-breaking change which adds functionality)

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

**Comment by blackenjoy27** [comment]
Ahmed and I work on this together and it cause no conflict, and it also resolve the problem we were working on. Thus, I approved the pull request.

**Review by blackenjoy27 — COMMENTED** [review]
Ahmed and I work on this together and it cause no conflict, and it also resolve the problem we were working on. Thus, I approved the pull request.

File: src/state/actions/serviceActions.js [hunk 1]
```diff
@@ -122,8 +122,9 @@ export const getServiceProviders = () => dispatch => {
   dispatch({ type: GET_ALL_SERVICE_PROVIDERS_START });
 
   axiosWithAuth()
-    .get(`api/providers/getserviceproviders`)
+    .get(`api/providers`)
     .then(res => {
+      console.log('PROVIDERS', res.data);
```
Comment (LittleBoGeek) [inline src/state/actions/serviceActions.js:127]: I would suggest removing this console.log in a future PR

File: src/components/common/ServicesTable/ServiceTable.js [hunk 2]
```diff
@@ -467,6 +470,8 @@ const ServicesTable = ({
 
   return (
     <div className="servicesTable">
+      {console.log('LOOK HERE', services)}
```
Comment (LittleBoGeek) [inline src/components/common/ServicesTable/ServiceTable.js:473]: I would also suggest removing these two console.logs in a future PR.

**Review by LittleBoGeek — COMMENTED** [review]
Overall, the code looks and runs well so fantastic job on this!

**Pull request merged**